### DISCUSSION TITLE
[Feature Request] Regex support for domain in dynamic filtering rules

### DISCUSSION THREAD

**Discussion by hotfur** [uBlockOrigin/uBlock-issues#3505] [General]
Hi everyone,

I enabled hard mode but want to allow `example[0-9].com`, so theoretically I would want to add `* example[0-9].com * noop` as a dynamic filtering rule but regex would not work. Any workaround?

**Comment by gorhill** [comment]
No workaround, and no plan to change this. Dynamic filtering is implemented using lookup tables and I don't plan on ever changing this.

**Comment by ghost** [comment]
Duplicate https://github.com/uBlockOrigin/uBlock-issues/issues/2242 and https://github.com/uBlockOrigin/uBlock-issues/issues/439
